ros_bootstrap.sh

```diff
@@ -3,7 +3,7 @@ set -xe
 
 # Set up ROS APT and install basic dependencies (rosdep, rosinstall). Must have ROS_VERSION set when called.
 sudo apt-get update && sudo apt-get install -q -y dirmngr curl gnupg2 lsb-release zip python3-pip python3-apt dpkg
-sudo -H pip3 install -U setuptools!=50.0.0
+# sudo -H pip3 install -U setuptools!=50.0.0
```

Comment:
When finalizing the PR, let's remove this line instead of commenting it out.

Reply:
Done